### PR TITLE
Fix Termux, better UI, add flatpak and snap

### PR DESCRIPTION
**Pull request by archetipico**
Title and fixing #25

### CODE REVIEW

File: src/other/global.sh [hunk 1]
```diff
@@ -1,8 +1,13 @@
 #!/bin/bash
+
+GREEN="\e[1;32;49m"
+NORMAL="\e[0m"
+RED="\e[1;91;49m"
```
Comment (MasterCruelty) [inline src/other/global.sh]: Ok I think that this solution definitely solves the problem with termux is solved. :ok_hand:

File: src/other/trash.sh [hunk 2]
```diff
@@ -0,0 +1,19 @@
+#!/bin/bash
+
```
Comment (MasterCruelty) [inline src/other/trash.sh]: A source file for trash was my next step, so it's ok.

**Review by MasterCruelty — APPROVED** [review]
:eyeglasses: reviewed.
snap and flatpak added to features of updater

**Comment by archetipico** [comment]
The WoW beta version

**Comment by MasterCruelty** [comment]
A man of cuture 🤓

**Pull request merged**